kubectl-pg/cmd/check.go

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
```

Comment:
we will clarify the question with licences and keep you informed

Reply:
Okay till then I will remove the license related content.

Reply:
@VineethReddy02 this is a legal document; nobody can remove them freely. If the library requires the resultant code to stay under the Apache license, it must stay under this licence

Reply:
Okay @sdudoladov. I will not remove anything generated by cobra library regarding the license.